registry/regsrc/module.go

```diff
@@ -110,23 +115,23 @@ func ParseModuleSource(source string) (*Module, error) {
 }
 
 // Display returns the source formatted for display to the user in CLI or web
-// output.
+// output. This always includes a hostname.
```

Comment:
AFAIK this didn't really change. I just clarified the comment to make it clear that the host is included, even if it was inferred. (though maybe I already diverged too far from the original)

Reply:
Ah yeah - it's confusing because `normalizeHostPrefix` will actually _remove_ the hostname even if it was originally given explicitly by the user in the case that it's the public registry's hostname for consistency.